src/api/payments/index.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import Resource from '../resource';
+import type { APIPromise } from '../resource';
+
+type GetCurrentBalanceParams = {
+    organization_id: number
+};
+
+type AddFundsParams = {
+    organization_id: number,
+    stripe_token: string,
+    amount: number,
+}
+
+export default class Payments extends Resource {
+    /**
+     * @api {get} /v1/organizations/:organization_id/payments get
+     * @apiGroup Payments
+     * @apiName getCurrentBalance
+     * @apiDescription Get the organization's current balance
+     * @apiParam {Number} organization_id
+     * @apiExample {js} Example:
+     *             gigwalk.payments.getCurrentBalance({...})
+     */
+    getCurrentBalance(params: GetCurrentBalanceParams): APIPromise<[{ current_balance: number }]> {
+        return this.client.get(`/v1/organizations/${params.organization_id}/payments`);
+    }
+
+    /**
+     * @api {post} /v1/organizations/:organization_id/payments post
+     * @apiGroup Payments
+     * @apiName addFunds
+     * @apiDescription Add funds to an organization's account
+     * @apiParam {Number} organization_id
+     * @apiParam {String} stripe_token
+     * @apiParam {Number} amount
+     * @apiExample {js} Example:
+     *             gigwalk.payments.addFunds({...})
+     */
+    addFunds(params: AddFundsParams): APIPromise<void> {
+        const { organization_id: organizationId, ...data } = params;
```

Comment:
I've started using this approach to gather the rest of the params into a single object. This means you can add extraneous data in the payload, but it should be relatively harmless. Either the API will respond with marshmallow schema errors, or the extra params will be ignored.

One benefit is that if the BE adds an extra field to some endpoint, then we don't need to update the library